### PR TITLE
bump sdk to 232

### PR DESCRIPTION
**Pull request by van800**
internal ci merge started https://buildserver.labs.intellij.net/buildConfiguration/DotNetDiv_Wave232_Service_SubtreePluginsSync_Unity_TriggerMergeBranchFromSubtree?branch=net232-sdk-update&mode=builds

### CODE REVIEW

File: debugger/debugger-worker/src/debugger.csproj [hunk 1]
```diff
@@ -19,7 +19,7 @@
     <Compile Include="..\..\..\resharper\build\generated\Model\DebuggerWorker\**\*" LinkBase="Model" Condition="!$(InternalBuild)" />
   </ItemGroup>
   <ItemGroup>
-    <PackageReference Include="Microsoft.NETFramework.ReferenceAssemblies.net472">
+    <PackageReference Include="Microsoft.NETFramework.ReferenceAssemblies" Version="1.0.3">
```
Comment (citizenmatt) [inline debugger/debugger-worker/src/debugger.csproj:22]: Why has this lost the `.net472` bit?

Comment (citizenmatt) [reply debugger/debugger-worker/src/debugger.csproj:22]: I think this also needs to be added to `PackagesLock.targets`, or we'll get errors in the monorepo

**Review by citizenmatt — APPROVED** [review]
Looks good to me, but some minor feedback.

File: debugger/debugger-worker/src/debugger.csproj (hunk 1 above)
Comment (van800) [reply debugger/debugger-worker/src/debugger.csproj:22]: This change fixes error
```
       /home/runner/work/resharper-unity/resharper-unity/resharper/resharper-yaml/src/Yaml.Rider/Yaml.Rider.csproj : error NU1605:  JetBrains.ReSharper.Plugins.Yaml.Rider -> Microsoft.NETFramework.ReferenceAssemblies 1.0.3 -> Microsoft.NETFramework.ReferenceAssemblies.net472 (>= 1.0.3)  
```
see https://github.com/JetBrains/resharper-unity/actions/runs/4576761611/jobs/8081322066

**Pull request closed without merging**